operator/pkg/lib/sidecars/pods/filter.go

```diff
@@ -0,0 +1,133 @@
+package pods
+
+import (
+	"encoding/json"
+	v1 "k8s.io/api/core/v1"
+)
+
+const AnnotationResetWarningKey = "istio.reconciler.kyma-project.io/proxy-reset-warning"
```

Comment:
As far as I remember we don't want to annotate customer workload any longer. That means that the restart won't set it and therefore we don't need to use it as a filter.

Reply:
Removed

Reply:
Discussion outcome: We can remove it.